source/xcontrol.rst

```diff
@@ -345,72 +345,154 @@ The detailed input looks like
 
 .. code:: text
 
-  $cma
   $wall
      potential=logfermi
      sphere: auto, all
   $end
 
-You can be more precise on the radius by giving the value in bohr instead
-of ``auto``. I personally recommend to use the logfermi potential, since it
-is best suited for confinements, but yet not the default.
+You can be more precise on the radius by giving the value in Bohr instead
+of ``auto``. The automatically determined radius is based on the largest
```

Comment:
The largest distance from the cma along the axis?

Reply:
The largest interatomic distance is currently used.